docs/answerfile.md

```diff
@@ -237,12 +237,18 @@ If `protov6` is static then the following elements must be present:
 
 #### Root password
 
-Specifies the root password. The value `!!` and a type of "hash" defers setting a password until first boot. Default: type="hash", `!!`.
+Specifies the root password. The value `!!` and a SHA-512 "hash" defers setting a password until first boot. Default: type="hash", `!!`.
```

Comment:
I must insist :)

"hash" here designates the string H A S H. The precision about SHA-512 does not belong here. Let's just remove it, as your command snippet below is rather obvious about SHA-512 being used.

```suggestion
Specifies the root password. The value `!!` and a type of  "hash" defers setting a password until first boot. Default: type="hash", `!!`.
```